tests/sentry/grouping/snapshots/test_variants/test_event_hash_variant/mobile@2021_02_12/group_125_event_126.pysnap

```diff
@@ -18,7 +18,7 @@ app-depth-1:
           frame*
             function*
               "stripped_application_code"
-        type (ignored because exception is synthetic)
+        type (ignored because type is platform-specific)
```

Comment:
Meta discussion: Isn't the reason we ignore the type here is not because it's platform-specific, but rather because it indicates a synthetic stack trace?

Reply:
The type is both platform-specific (or at least I defined it to be) and with a synthetic stacktrace. Only one reason is stored, last write wins. This is something we could change easily if we wanted to, it does not affect grouping, just the debugger.

Reply:
Sorry for being unclear, that difference is what I'm trying to get to. The regexes aren't trying to find platform-specific types, but rather types that indicate synthetic exceptions.

If we truly wanted to "fix" platform-specific types, we would map them to unique, platform-independent representations. But in this case, we want to ignore them since they are synthetic exceptions.